pyproject.toml

```diff
@@ -54,11 +54,11 @@ classifiers=[
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<4.0"
+python = ">=3.9,<3.13"
```

Comment:
Whats the reason for this upper bound set to <3.13?

Reply:
3.13 is the highest python version that currently exists. Poetry complained that some of the dependencies added (it was types for pandas, I think) were not compatible with python versions greater than 3.13. So I adjusted the range to reflect the current reality.